### PR TITLE
Wave2: Adding docs for 11 more pack errors and warnings

### PR DESCRIPTION
**Pull request by mishra14**
Working towards: https://github.com/NuGet/Home/issues/6498

This PR adds docs for 11 of the pack errors and warnings. More to come in the next wave

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5022?branch=pr-en-us-1009) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5000.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5000.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5000?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5002.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5002.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5002?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5003.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5003.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5003?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5004.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5004.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5004?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5008.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5008.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5008?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5009.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5009.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5009?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5010.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5010.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5010?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5011.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5011.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5011?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5012.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5012.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5012?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5013.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5013.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5013?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5014.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5014.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5014?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5015.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5015.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5015?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5016.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5016.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5016?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5017.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5017.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5017?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5018.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5018.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5018?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5019.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5019.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5019?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5022.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5022.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5022?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5023.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5023.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5023?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5024.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5024.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5024?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5025.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5025.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5025?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5026.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5026.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5026?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5027.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5027.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5027?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5028.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5028.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5028?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5029.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5029.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5029?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5100.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5100.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5100?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5101.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5101.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5101?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5102.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5102.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5102?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5103.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5103.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5103?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5104.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5104.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5105.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5105.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5105?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5115.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5115.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5115?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5500.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5500.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5500?branch=pr-en-us-1009) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C9%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201808090046237114-1009%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Uhc0LQXAogUrGkeFmRhn08sReE9CjEyr5Dqff2Glxkg%3D&st=2018-08-09T00%3A42%3A30Z&se=2018-09-09T00%3A47%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/errors-and-warnings/NU5104.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+manager: rrelyea
+ms.date: 8/8/2018
+ms.topic: reference
+ms.reviewer: anangaur
+f1_keywords:
+  - NU5104
+---
+
+# NuGet Warning NU5104
+<pre>A stable release of a package should not have a prerelease dependency. Either modify the version spec of dependency "NuGet.Versioning [4.7.0-preview4.5065, )" or update the version field in the nuspec.</pre>
```
[the remark this answers is not in the document]
Comment (mishra14) [reply docs/reference/errors-and-warnings/NU5104.md]: What do you mean by change in the code?

[the remark this answers is not in the document]
Comment (nkolev92) [reply docs/reference/errors-and-warnings/NU5104.md]: What would the message be when the client hits this warning? 

I'd imagine both of these are the same.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5022?branch=pr-en-us-1009) |
[docs/reference/Errors-and-Warnings.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/Errors-and-Warnings.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5022.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5022.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5022?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5023.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5023.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5023?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5024.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5024.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5024?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5025.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5025.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5025?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5027.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5027.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5027?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5028.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5028.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5028?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5029.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5029.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5029?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5102.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5102.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5102?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5103.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5103.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5103?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5104.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5104.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104?branch=pr-en-us-1009) |
[docs/reference/errors-and-warnings/NU5105.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-anmishr-packerrordocs2/docs/reference/errors-and-warnings/NU5105.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5105?branch=pr-en-us-1009) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201808141832313305-1009%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=jjxi9oIZLTer%2FrCColdWYEo4DHK9QXf%2FKEMm6y2YZxg%3D&st=2018-08-14T18%3A29%3A06Z&se=2018-09-14T18%3A34%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**